pytest_arch/rule.py

```diff
@@ -118,10 +118,10 @@ def may_import(self, pattern):
         self.ignored.append(pattern)
         return self
 
-    def check(self, package: str | ModuleType):
+    def check(self, package: str | ModuleType, *, lazy_imports=True):
```

Comment:
if you call it `only_toplevel` at in the other functions and here `only_toplevel_imports` the naming is consistent

Reply:
(the other functions have `import` in the name, but `check()` doesn't